server/app/views/style/BaseStyles.java

```diff
@@ -185,11 +185,7 @@ public final class BaseStyles {
 
   public static final String ADMIN_LOGIN =
       StyleUtils.joinStyles(
-          "bg-transparent",
-          "text-black",
-          "underline",
-          "font-bold",
-          StyleUtils.hover("bg-gray-200", "opacity-90"));
+          "text-black", "underline", "font-bold", StyleUtils.hover("bg-gray-200", "opacity-90"));
```

Comment:
Why: The new ordering in the css file causes `bg-transparent` to load later now upsetting the visuals.